package.json

```diff
@@ -50,6 +50,7 @@
     "react-test-renderer": "16.8.6",
     "redux": "4.0.1",
     "rimraf": "2.6.3",
+    "sass": "^1.23.3",
```

Comment:
Should we pin the version?

Reply:
Yes, I forgot that we usually ignore `package-lock.json` and instead fix the version in `package.json`. So I fixed that also in b22a677c8 and additionally added `save-exact = true` in my `˜/.npmrc` so this hopefully does not happen again.

With this, this PR is up for a review again.